src/TEOS10.jl

```diff
@@ -183,7 +183,7 @@ approximation to TEOS-10 described in Roquet et al. (§3.1, 2014).
   for the density and specific volume of seawater using the TEOS-10 standard. Ocean Modelling.
 """
 @inline ρ(Θ, Sᴬ, Z, ::EOS₁₀) = _ρ(τ(Θ), s(Sᴬ), ζ(Z))
-@inline ρ′(Θ, Sᴬ, Z, eos::EOS₁₀) = ρ(Θ, Sᴬ, Z, eos) - eos.reference_density 
+@inline ρ′(Θ, Sᴬ, Z, eos::EOS₁₀) = r'(τ(Θ), s(Sᴬ), ζ(Z)) - eos.reference_density 
```

Comment:
`r'(τ(Θ), s(Sᴬ), ζ(Z))` - > `r(τ(Θ), s(Sᴬ), ζ(Z))` as per equation (13) in [Roquet et al. 2015](https://doi.org/10.1016/j.ocemod.2015.04.002) 

$$
\rho(S_{A}, \Theta, z) = r_{0}(z) + r(S_{A}, \Theta, z)
$$

Reply:
I think `r′` here is Roquet's `r`. Is that what you are saying?

Reply:
Yep exactly.

Reply:
If you prefer `r′` to `r` that is fine.

Reply:
Sorry just to be clear here I meant changing the name of the function `r′` to `r` to match equation (13) from Roquet et al (2015) above. Then keep everything else the same so that https://github.com/CliMA/SeawaterPolynomials.jl/blob/c8198d3003b9e4987b83789c1dc14a83e3ce41a2/src/TEOS10.jl#L188C1-L188C42 becomes

```julia
@inline _ρ(τ, s, ζ) = r₀(ζ) + r(τ, s, ζ)
```

to more closely equation (13). Not necessary and will likely cause something else to change to probably best leave as is!